mkdocs-en.yml

```diff
@@ -43,16 +43,15 @@ nav:
   - Analysis:
       - Kubeflow: 1-Experiments/Kubeflow.md
       - Jupyter: 1-Experiments/Jupyter.md
-      - ML-Workspaces: 1-Experiments/ML-Workspaces.md
+      - ML-Workspaces: 1-Experiments/Virtual-Desktop.md
```

Comment:
Have we adopted the virtual desktop nomenclature?  If so, we should change the heading entirely.

Reply:
Oh yeah! I changed the H1 in the document, but forgot about the menu key. I have a couple minor fixes to push at some point, so I'll wrap this into that PR.